[FR] Add `Mapping and Modding Timeline`, fix links in other file (#5652)
ppy/osu-wiki: wiki/Mapping_and_Modding_Timeline/fr.md

  - Le leadership est passé provisoirement entre les deux utilisateurs jusqu'à ce que ztrot prenne le contrôle en février 2011.

### Octobre

- **03/10/2010 :** Les MAT ont été autorisés à proposer des beatmaps pour le classement avec des icônes "bubble" normales (![icône bubble](/wiki/shared/icon/bubble.gif)), par opposition aux proto-bulles.
  - Les proto-bulles étaient ignorées par la plupart des membres du BAT, et le MAT n'avait donc aucune utilité sans ce changement.
  - Cela a donné aux MAT beaucoup plus d'influence dans le processus de classement.

### December

- **31/12/2010 :** ![][flag_US] [Derekku](https://osu.ppy.sh/users/91341) a été nommé chef des BAT après la démission de ![][flag_SG] [Pasonia](https://osu.ppy.sh/users/43345).
  - Comme ses prédécesseurs, il a continué à s'occuper de la promotion des nouveaux BAT et à maintenir les BAT actuelles.

**1864 beatmaps** ont été classés en 2010.

## 2011

### Mars

- **09/03/2011 :** Les membres du MAT ont reçu l'autorisation de mettre des bubbles sur des beatmaps de la catégorie [approuvée](/wiki/Beatmap/Category#approuvée).

>   - Seule une des deux bubble d'une carte approuvée pouvait être placée par un membre du MAT.

Reviewer: Je pense qu'il est préférable de rester sur "beatmap" même s'il n'y a que "map" dans la version originale, à moins que l'on prenne aussi "map" comme abrévation de "beatmap"
```suggestion
  - Seule une des deux bubble d'une beatmap approuvée pouvait être placée par un membre du MAT.
```

>   - Ce changement a été mis en œuvre parce que les beatmaps approuvés devenaient plus courants.

Reviewer: ```suggestion
  - Ce changement a été mis en œuvre parce que les beatmaps approuvées devenaient plus courantes.
```

- **23/08/2011 :** La "Beatmap Approval Team" a été renommé en "Beatmap Appreciation Team".
  - Le but de ce nouveau nom était d'unir les utilisateurs impliqués dans l'approbation des beatmaps et les utilisateurs impliqués dans la modération générale (essentiellement la [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team)) sous le même nom, que "Beatmap Approval Team" ne représentait pas efficacement.

### Octobre

- **08/10/2011 :** Le sous-forum [Ranking Criteria](https://osu.ppy.sh/community/forums/87) est ouvert à la discussion publique sur les propositions des [critères de classement](/wiki/Ranking_Criteria).
  - Ces discussions étaient auparavant tenues par les membres du BAT dans un sous-forum privé.
- **10/10/2011 :** Les membres du MAT n'étaient plus autorisés à nommer ou à voter pour les nouveaux membres de leur équipe.
  - Ayant obtenu des autorisations pour placer des bubbles, il a été supposé que le MAT voterait pour son propre intérêt, ce qui a été empêché avant que des dommages ne soient causés.

>   - Les BAT a acquis l'entière responsabilité des ajouts des MAT.

Reviewer: ```suggestion
  - Les BAT ont acquis l'entière responsabilité des ajouts des MAT.
```

> **1368 beatmaps** ont été classés en 2011. Il s'agit de la première baisse après 4 années de croissance consécutives.

Reviewer: ```suggestion
**1368 beatmaps** ont été classées en 2011. Il s'agit de la première baisse après 4 années de croissance consécutives.
```

## 2012

### Juillet

- **22/07/2012 :** La première itération des [points de performance](/wiki/Performance_Points) connue sous le nom de [ppv1](/wiki/Performance_Points/ppv1) a été mise en place, remplaçant les classements par score total classé.

### Août

> - **21/08/2012 :** Les beatmaps de la catégorie [approuvée](/wiki/Beatmap/Category#approved) ont commencé à récompenser le score classé.

Reviewer: Je pense qu'on peut trouver mieux mais en tout cas ici la traduction est trop littérale
```suggestion
- **21/08/2012 :** Les beatmaps de la catégorie [approuvée](/wiki/Beatmap/Category#approved) ont commencé à faire gagner du score classé.
```

>   - Le score classé n'est plus utilisé comme principale mesure de score, de sorte que les beatmaps approuvés ne peuvent plus être utilisés abusivement pour le classement des joueurs.

Reviewer: ```suggestion
  - Le score classé n'est plus utilisé comme principale mesure de score, de sorte que les beatmaps approuvées ne peuvent plus être utilisées abusivement pour le classement des joueurs.
```

> **1460 beatmaps** ont été classés en 2012.

Reviewer: ```suggestion
**1460 beatmaps** ont été classées en 2012.
```

## 2013

### Mars

- **20/03/2013 :** Les BAT ont été divisés en BAT et en membre de la [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) (*GMT*).
  - De nombreux membres de la BAT étaient présent uniquement pour les fonctions de modération, ce qui a permis de clarifier les rôles de modding et de membres modérateurs.
  - Les membres du nouveau GMT étaient auparavant connus sous le nom de "Green BAT".

### Avril

> - **23/04/2013 :** Le MAT a été démantelé.

Reviewer: Vu que y a "la MAT" juste après et que ça me semble plus logique de dire "la" vu que ça désigne une équipe.
```suggestion
- **23/04/2013 :** La MAT a été démantelée.
```

  - Tous les membres de la MAT ont été ajoutés aux BAT.
  - Ce changement a été effectué parce que la frontière entre les MAT et les BAT était devenue trop mince. Un membre de la MAT était en fait un membre de la BAT qui ne pouvait pas classer les beatmaps, ce qui ne correspondait pas aux objectifs initiaux des MAT en tant que nouveau groupe d'utilisateurs.

>   - Les BAT était également assez inactive à cette époque, un afflux important de nouveaux membres a donc permis de résoudre ce problème.

Reviewer: Ici "les BAT" sous-entends "les membres de la BAT" donc masculin.
```suggestion
  - Les BAT étaient également assez inactifs à cette époque, un afflux important de nouveaux membres a donc permis de résoudre ce problème.
```

### Mai

- **15/05/2013 :** Le [Triumvir Conglomerate](/wiki/Modding/BAT_Managers#triumvir-conglomerate) a été formé.
  - Le rôle de gestionnaire de la BAT a été partagé entre 3 utilisateurs de différentes régions afin de promouvoir une contribution égale entre les sous-groupes de mapping. ![][flag_US] [Garven](https://osu.ppy.sh/users/244216) représentait l'Amérique du Nord, ![][flag_CN] [NatsumeRin](https://osu.ppy.sh/users/151679) l'Asie, et ![][flag_FR] [Kurai](https://osu.ppy.sh/users/77089) l'Europe.
  - Avant cela, les sous-groupes asiatiques étaient souvent négligés car ils communiquaient principalement en dehors de l'IRC/des forums d'osu!
  - Lorsque les 3 membres n'étaient pas d'accord sur une décision, celle-ci devait être décidée par un supérieur (le plus souvent ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335)).

### Août

- **01/08/2013 :** La politique de déclassement des beatmaps a été mise en place, permettant aux membres du BAT de déclasser un beatmap dans la semaine suivant son classement. 
  - Ce changement était un test pour la catégorie [qualifiée](/wiki/Beatmap/Category#qualifiée) qui a été implémentée plus tard.

> - **02/08/2013 :** Les premier BAT du mode osu!taiko ont été ajoutées, ce qui signifie une séparation des autorisations de mode de jeu dans le groupe.

Reviewer: ```suggestion
- **02/08/2013 :** Les premiers BAT du mode osu!taiko ont été ajoutées, ce qui signifie une séparation des autorisations de mode de jeu dans le groupe.
```

  - Avant cette date, n'importe quel BAT pouvait théoriquement proposer des beatmaps de n'importe quel mode, mais la croissance des différents modes de jeu a rendu cette distinction nécessaire.

### Décembre

> - **12/12/2013 :** Les beatmaps avec deux nominations ont commencé à être promus dans la catégorie [qualifiée](/wiki/Beatmap/Category#qualifiée) au lieu de la catégorie classée.

Reviewer: ```suggestion
- **12/12/2013 :** Les beatmaps avec deux nominations ont commencées à être promues dans la catégorie [qualifiée](/wiki/Beatmap/Category#qualifiée) au lieu de la catégorie classée.
```

>   - Les beatmaps qualifiés pourraient être disqualifiés dans un délai d'une semaine, remplaçant ainsi la possibilité de déclasser un beatmap dans un délai d'une semaine. 

Reviewer: ```suggestion
  - Les beatmaps qualifiées pourraient être disqualifiées dans un délai d'une semaine, remplaçant ainsi la possibilité de déclasser une beatmap dans un délai d'une semaine. 
```

- **22/12/2013 :** Le [Triumvir Conglomerate](/wiki/Modding/BAT_Managers#triumvir-conglomerate) fait tourner ses dirigeants régionaux pour éviter l'épuisement.
  - ![][flag_US] [Charles445](https://osu.ppy.sh/users/85000) représentait l'Amérique du Nord, ![][flag_CN] [popner](https://osu.ppy.sh/users/759860) l'Asie, et ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) l'Europe.

> **1327 beatmaps** ont été classés en 2013.

Reviewer: ```suggestion
**1327 beatmaps** ont été classées en 2013.
```

## 2014

### Février

- **10/02/2014 :** Pour améliorer l'organisation, le système de modding et de mapping, la gestion interne est passée des forums à Trello.
- **18/02/2014 :** La catégorie [approuvée](/wiki/Beatmap/Category#approuvée) a été retirée.
  - Les beatmaps marathon ont été promus dans la catégorie classée avec 3 nominations BAT (2 icônes `bubble` (![icône bubble](/wiki/shared/icon/bubble.gif)) + 1 icône `flame` (![icône flame](/wiki/shared/icon/flame.gif))).

### Août

> - **21/08/2014 :** La [Quality Assurance Team](/wiki/Modding/Quality_Assurance_Team) (*QAT*) se sépare des BAT actuelle.

Reviewer: ```suggestion
- **21/08/2014 :** La [Quality Assurance Team](/wiki/Modding/Quality_Assurance_Team) (*QAT*) se sépare des BAT actuels.
```

>   - Les BAT (souvent appelé newBAT à cette époque) désigne les beatmaps pour le classement.

Reviewer: ```suggestion
  - Les BAT (souvent appelé newBAT à cette époque) désigne les beatmaps à classer.
```

>   - Les QAT s'assure que les beatmaps qualifiés étaient de qualité suffisante. Les beatmaps qui ne répondaient pas aux normes de qualité étaient disqualifiés.

Reviewer: ```suggestion
  - Les QAT s'assure que les beatmaps qualifiés étaient de qualité suffisante. Les beatmaps qui ne répondaient pas aux normes de qualité étaient disqualifiées.
```

  - Cela a essentiellement inversé la fusion des BAT et des MAT l'année dernière.
  - La division a été mise en œuvre dans le cadre d'un essai manuel pour le [modding v2](/wiki/Beatmap_Discussion) qui tente d'automatiser la promotion et la retraite des BAT.

>   - Malgré les tentatives de présenter les disqualifications comme un changement positif pour la qualité des maps, la communauté les a perçues de manière négative. Cela était particulièrement évident dans le cas de beatmaps controversés comme [disqualification on *Halozy - Genryuu Kaiko*](https://osu.ppy.sh/community/forums/posts/3591479)

Reviewer: ```suggestion
  - Malgré les tentatives de présenter les disqualifications comme un changement positif pour la qualité des maps, la communauté les a perçues de manière négative. Cela était particulièrement évident dans le cas de beatmaps controversées comme [disqualification on *Halozy - Genryuu Kaiko*](https://osu.ppy.sh/community/forums/posts/3591479)
```

### Octobre

- **15/10/2014 :** Le sous forum [Beatmap Management](https://osu.ppy.sh/community/forums/115) a remplacé Trello.

>   - L'accessibilité du public aux BAT a augmenté, ce qui était l'objectif du modding v2.

Reviewer: ```suggestion
  - L'accessibilité du public aux BAT a augmentée, ce qui était l'objectif du modding v2.
```

>   - Les questions relatives aux beatmaps nominés/qualifiés ont été postées ici, ainsi que toute annonce relative aux systèmes de mapping/modding.

Reviewer: ```suggestion
  - Les questions relatives aux beatmaps nominées/qualifiées ont été postées ici, ainsi que toute annonce relative aux systèmes de mapping/modding.
```

### Novembre

- **14/11/2014 :** La promotion des nouveaux BAT s'est faite par le biais de votes publics parmi les BAT/QAT actuels.

>   - Les utilisateurs ayant obtenu au moins 15 votes et dont l'activité de modding est calculée automatiquement seront promus au groupe par intermittence.

Reviewer: ```suggestion
  - Les utilisateurs ayant obtenu au moins 15 votes et dont l'activité de modding est calculée automatiquement seraient promus au groupe par intermittence.
```

  - Cela a été géré sur Trello dans le passé.

> **1381 beatmaps** ont été classés en 2014.

Reviewer: ```suggestion
**1381 beatmaps** ont été classées en 2014.
```

## 2015

### Février

> - **01/02/2015 :** Les noms d'utilisateurs rouges ont été supprimés des BAT et les BAT a été renommé en [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BN*).

Reviewer: ```suggestion
- **01/02/2015 :** Les noms d'utilisateurs rouges ont été supprimés des BAT et les BAT ont été renommés en [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BN*).
```

  - Le groupe était géré par ![][flag_DE] [Okoratu](https://osu.ppy.sh/users/1623405) et ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418).
  - Le nom de l'UBKRC a été choisi parce qu'il était ridiculement stupide. Son but était de paraître moins autoritaire que le conseil des critères de classement.

### Août

- **05/08/2016 :** Le [blog Quality Assurance Team](/wiki/Quality_Assurance_Team_Blog) a été créé.
  - Les QAT était critiquée pour être trop fermée au reste de la communauté de mapping. Ce blog a donc été créé pour accroître la transparence des décisions des QAT et des activités de la communauté.
- **26/08/2016 :** Le [Code de conduite](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping) a été mis en place pour définir les attentes en matière de comportement des moddeurs.
  - Jusqu'à présent, ces règles et directives étaient considérées comme relevant du bon sens. En les documentant, la modération pourrait être rendue plus cohérente.

### Septembre

- **30/09/2016 :** Le [Community Mentorship Program](/wiki/Community_Mentorship_Program) a été relancé après une longue interruption.

>   - Sa nouvelle structure était centrée sur la communication par [Discord](https://discord.com/brand-new) et les cycles saisonniers.

Author: ```suggestion
  - Sa nouvelle structure était centrée sur la communication par [Discord](https://discord.com/) et les cycles saisonniers.
```

  - Le programme était dirigé par ![][flag_DE] [Okoratu](https://osu.ppy.sh/users/1623405) et ![][flag_AR] [Yuii-](https://osu.ppy.sh/users/2935923)

### Octobre

- **17/01/2016 :** La catégorie [Loved](/wiki/Beatmap/Category#loved) a été créée.
  - Cette catégorie fournissait des classements pour les maps populaires qui ne pouvaient pas être classées pour diverses raisons.
  - *Voir [History of Loved](/wiki/Beatmap/History_of_Loved) pour une chronologie détaillée des changements du système Loved.*

### Novembre

- **01/11/2016 :** Le système [Beatmap Veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) a été mis en place.
  - Les Beatmap Nominators pouvaient arrêter une beatmap dans le processus de classement pour des raisons subjectives en réinitialisant une nomination. Ceci était fait avec une icône `bubble pop` (![icône bubble pop](/wiki/shared/icon/bubble-pop.gif)).

>   - Les vétos pourraient être écrasés par les nominations des Beatmap Nominators qui n'ont pas été impliqués dans la beatmap.

Reviewer: ```suggestion
  - Les vétos pouvaient être écrasés par les nominations des Beatmap Nominators qui n'ont pas été impliqués dans la beatmap.
```

  - Afin de laisser suffisamment de temps pour opposer son veto, les nominations consécutives ne pouvaient pas être fixées à moins de 24 heures d'intervalle.
  - Les vetos non concluants ont fait l'objet d'une médiation par les QAT peu après la mise en œuvre de ce système.
- **02/11/2016 :** Les fichiers des Beatmap Nominator ont été créés pour garder un œil sur tous les membres individuels des BN.
  - Ces dossiers comprenaient les activités de modding/nomination, les disqualifications et les problèmes de comportement.
  - Les membres du BN ayant 3 "strikes" ou plus enregistrées dans ces fichiers seraient retirés du groupe.

> **1720 beatmaps** ont été classés en 2016.

Reviewer: ```suggestion
**1720 beatmaps** ont été classées en 2016.
```

## 2017

### Avril

- **06/04/2017 :** Les [Beatmap Nominator Tiers](/wiki/Modding/Beatmap_Nominator_Tiers) ont été implémentés, divisant les BN en deux sous-groupes appelés "tiers".
  - Comme le QAT ne jouait plus son rôle de contrôle de la qualité, ce système a été mis en place pour améliorer la qualité des beatmaps entrant dans la catégorie classée.
  - Ce système a imité le stade ultérieur de la [Mapping Assistance Team](/wiki/Modding/Mapping_Assistance_Team).

>     - Les nominateurs de niveau 1 pourraient mettre des bubbles sur les maps.

Reviewer: ```suggestion
    - Les nominateurs de niveau 1 pouvaient mettre des bubbles sur les maps.
```

>     - Les nominateurs de niveau 2 pourraient mettre des bubbles et qualifier les maps..

Reviewer: ```suggestion
    - Les nominateurs de niveau 2 pouvaient mettre des bubbles et qualifier les maps..
```

  - Aucun Beatmap Nominators n'a été ajouté au 2e niveau pendant les 5 mois où ce système était actif, ce qui a considérablement ralenti la vitesse de classement.

### Mai

- **22/05/2017 :** Le serveur Discord Beatmap Nominator a été créé pour les sous-divisions des BN.
  - Les sous-divisions ont été conçues comme des groupes de discussion composés de nominateurs sélectionnés au hasard dans les deux niveaux, dirigés par un membre de la QAT.
  - La plupart des responsables des QAT sont passés à ce serveur malgré la résistance des hauts responsables d'osu!
- **23/05/2017 :** Le bot Discord [Aiess](/wiki/Projets#miscellaneous) a été créé par ![][flag_SE] [Naxess](https://osu.ppy.sh/users/8129817).
  - Il s'agit d'un flux d'événements liés aux mapping sur les serveurs Discord d'osu! en commençant par les flux de nomination et de disqualification.

### Juin

- **02/06/2017 :** Les beatmaps marathons ont commencé à exiger 2 nominations au lieu de 3.

>   - Auparavant, il fallait 3 nominations car on s'attendait à ce que les marathons aient plus de contenu à vérifier qu'une beatmap normal, mais ce n'est plus vrai.

Reviewer: ```suggestion
  - Auparavant, il fallait 3 nominations car on s'attendait à ce que les marathons aient plus de contenu à vérifier qu'une beatmap normale, mais ce n'est plus vrai.
```

  - L'icône "flame" (![icône flame](/wiki/shared/icon/flame.gif)) n'est plus utilisée.

### Septembre

> - **10/09/2017 :** Le "[QAT Upheaval](https://osu.ppy.sh/community/forums/topics/635507)" a été mis en place. Il s'agissait d'une série de changements en réponse au mécontentement de la communauté de mapping sous la direction de l'[osu!team](/wiki/People/The_Team). Suite à ce changement, les QAT a commencé à s'autogérer sans l'intervention de ses supérieurs.

Reviewer: ```suggestion
- **10/09/2017 :** Le "[QAT Upheaval](https://osu.ppy.sh/community/forums/topics/635507)" a été mis en place. Il s'agissait d'une série de changements en réponse au mécontentement de la communauté de mapping sous la direction de l'[osu!team](/wiki/People/The_Team). Suite à ce changement, les QAT ont commencés à s'autogérer sans l'intervention de leurs supérieurs.
```

  - Les [BN Tiers](/wiki/Modding/Beatmap_Nominator_Tiers) ont été remplacés par les [Probationary Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators#probationary-beatmap-nominators). Deux membres des BN en probation ne pouvaient pas nommer la même beatmap et les utilisateurs ne pouvaient pas être en probation pendant plus de deux mois à la fois.

>   - Beatmap Nominators were given purple titles on the forums.

Reviewer: ```suggestion
  - Des titres violets ont été donnés aux Beatmap Nominators sur les forums.
```

  - ![][flag_DE] [Okoratu](https://osu.ppy.sh/users/1623405) et ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508) ont été nommés [responsables des QAT](/wiki/Modding/QAT_Leaders) par un vote des membres du BN/QAT. Leur responsabilité était de travailler avec l'équipe d'osu! pour maintenir les communautés de mapping et de modding.

>   - Des badges de profil Beatmap Nominator et Quality Assurance Team ont été créés. Cette récompense et d'autres récompenses pour les Beatmap Nominator ont été prévues pour remotiver l'équipe actuellement insatisfaite.

Reviewer: ```suggestion
  - Des badges de profil Beatmap Nominator et Quality Assurance Team ont été créés. Cette distinction et d'autres récompenses pour les Beatmap Nominator ont été prévues pour remotiver l'équipe actuellement insatisfaite.
```

>   - Avec une direction moins autoritaire et plus d'utilisateurs capables de nommer des beatmaps après ces changements, les normes de mapping ont cessé d'être strictement contrôlées. Des beatmaps controversés qui n'auraient probablement pas été classés sous l'ancien système (par exemple *[FELT - Rendezvous](https://osu.ppy.sh/beatmapsets/725171#osu/1541573)*) ont atteint un statut classé.

Reviewer: ```suggestion
  - Avec une direction moins autoritaire et plus d'utilisateurs capables de nommer des beatmaps après ces changements, les normes de mapping ont cessées d'être strictement contrôlées. Des beatmaps controversées qui n'auraient probablement pas été classées sous l'ancien système (par exemple *[FELT - Rendezvous](https://osu.ppy.sh/beatmapsets/725171#osu/1541573)*) ont atteint un statut classé.
```

### Octobre

- **01/10/2017 :** Les [BN tests](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Nominator_Test) ont été remplacés par l'examen du QAT des mods anonymes de chaque candidat BN.

>   - Ce changement a été effectué parce que la tricherie aux BN test était devenue impossible à éviter.

Reviewer: ```suggestion
  - Ce changement a été effectué parce que la tricherie aux BN test était devenue monnaie courante.
```

### Décembre

- **17/12/2017 :** ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) a cessé de s'impliquer dans l'équipe d'osu!, notamment en participant activement au QAT.

> **1847 beatmaps** ont été classés en 2017.

Reviewer: ```suggestion
**1847 beatmaps** ont été classées en 2017.
```

## 2018

### Février

- **08/02/2018 :** La [Mappers' Guild](/wiki/Mappers_Guild) a été ouverte aux candidatures publiques pour la création généralisée de beatmaps des [Artistes Associés](/wiki/Featured_Artists).

### Mars

- **01/03/2018 :** Les QAT  a séparé ses tâches en branches : évaluation des BE, disqualifications, métadonnées, spotlights, blog, critères de classement et direction générale.
  - Les membres des QAT peuvent être impliqués dans plusieurs branches.
  - L'objectif était de donner une raison d'être à certains membres du QAT qui estimaient ne pas avoir de rôle spécifique dans la communauté du mapping/modding. Ce sentiment s'explique en grande partie par le fait que la QAT n'est plus guère impliquée dans l'assurance qualité.
- **03/03/2018 :** Le canal `#modding` a été créé sur le [serveur Discord osu!dev](/wiki/osu!dev_Discord_server).
  - Alors que son objectif était de déplacer les discussions sur le mapping hors du serveur Discord des BN, ce centre de discussion était rarement utilisé en dehors des réunions programmées du QAT.

> - **24/03/2018 :** Le sous-forum Critères de classement a été rouvert pour les discussions publiques sur les propositions et l'UBKRC a été supprimé.

Reviewer: ```suggestion
- **24/03/2018 :** Le sous-forum Critères de classement a été rouvert pour les discussions publiques des propositions et l'UBKRC a été supprimé.
```

  - L'UBKRC a été interrompu à ce moment-là parce que toutes les sous-sections des critères de classement généraux et par mode de jeu ont été finalisées.

### Avril

> - **01/04/2018 :** La QAT a commencé à évaluer les demandes des BN en examinant les mods non anonymes, par opposition aux mods précédemment anonymes.

Reviewer: ```suggestion
- **01/04/2018 :** La QAT a commencée à évaluer les demandes des BN en examinant les mods non anonymes, par opposition aux mods précédemment anonymes.
```

  - Cette modification a été apportée car la QAT estimait que les mods anonymes hors contexte ne pouvaient pas déterminer efficacement les capacités d'un moddeur.

>   - Grâce à ce changement, la QAT a déterminé que la partialité ne pouvait être évitée lors de l'évaluation des Beatmap Nominators.

Reviewer: ```suggestion
  - Grâce à ce changement, la QAT a déterminée que la partialité ne pouvait être évitée lors de l'évaluation des Beatmap Nominators.
```

- **22/04/2018 :** La QAT a tenu sa première réunion sur le serveur osu!dev dans le but d'accroître la transparence des affaires de la QAT et d'éloigner les discussions internes du serveur Discord des BN.
  - Malgré cette tentative, la plupart des discussions internes étaient toujours traitées sur le serveur Discord des BN.

### Mai

> - **22/05/2018 :** ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) devient un [responsable QAT](/wiki/Modding/QAT_Leaders) après que ![][flag_DE] [Okoratu](https://osu.ppy.sh/users/1623405) s'est retiré.

Reviewer: ```suggestion
- **22/05/2018 :** ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) devient un [responsable QAT](/wiki/Modding/QAT_Leaders) après que ![][flag_DE] [Okoratu](https://osu.ppy.sh/users/1623405) se soit retiré.
```

  - Comme son prédécesseur, il a continué à travailler avec l'équipe d'osu! pour gérer les scènes de mapping/modding.

### Juin

- **21/06/2018 :** L'initiative [Quality Assurance Helper](/wiki/People/The_Team/Beatmap_Nominators/General_Information#quality-assurance-helpers) a été créée.
  - Cette mesure visait à garantir que tous les beatmaps qualifiés étaient examinés par au moins deux membres du BN ou de la QAT avant d'être classés.
  - Le programme a été géré via Trello et automatisé par ![][flag_SE] [Naxess](https://osu.ppy.sh/users/8129817) via Aiess.
  - Ce changement a été l'une des principales raisons pour lesquelles le QAT a été renommé par la suite.

### Juillet

- **09/07/2018 :** De nouvelles règles d'étalement des maps ont été mises en place, ce qui a complètement modifié la façon dont les difficultés sont créées dans les mapsets classés à l'avenir.

>   - Avant ce changement, toute carte dont le drain time était inférieur à 5 minutes nécessitait un écart de difficulté complet. Après ce changement, les exigences en matière d'écart de difficulté étaient plus souples pour les beatmaps plus longs dont le drain time était inférieur à 5 minutes.

Reviewer: ```suggestion
  - Avant ce changement, toute map dont le drain time était inférieur à 5 minutes nécessitait un écart de difficulté complet. Après ce changement, les exigences en matière d'écart de difficulté étaient plus souples pour les beatmaps plus longues dont le drain time était inférieur à 5 minutes.
```

  - *Pour plus de détails sur ce changement, lisez son [fil de discussion](https://osu.ppy.sh/community/forums/topics/726474).*

### Décembbre

> - **12/12/2018 :** L'équipe d'osu! a pris une [position négative sur les commissions des beatmap](https://osu.ppy.sh/community/forums/topics/840838).

Reviewer: ```suggestion
- **12/12/2018 :** L'équipe d'osu! a pris une [position négative sur les commissions des beatmaps](https://osu.ppy.sh/community/forums/topics/840838).
```

  - L'équipe reconnaît que des commissions se produiront, mais s'engage à ne prendre aucune mesure à moins que des membres du BN ou de la QAT soient impliqués.
- **17/12/2018 :** ![][flag_GB] [-Mo-](https://osu.ppy.sh/users/2202163) devient un [responsable QAT](/wiki/Modding/QAT_Leaders) après que ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508) s'est retiré.
  - Comme son prédécesseur, il a continué à travailler avec l'équipe d'osu! pour gérer les scènes de mapping/modding.
- **17/12/2018 :** La QA a été déclarée sans avenir dans son état actuel et les idées de réforme ont été encouragées.

>   - Cette décision a été prise parce que la QAT n'avait pas été impliqué dans quoi que ce soit lié à l'assurance qualité depuis des années, mais un système devait exister pour maintenir les Beatmap Nominators et la communauté de mapping en général.

Reviewer: ```suggestion
  - Cette décision a été prise parce que la QAT n'avait pas été impliquée dans quoi que ce soit lié à l'assurance qualité depuis des années, mais un système devait exister pour maintenir les Beatmap Nominators et la communauté de mapping en général.
```

  - À l'époque, la QAT suscitait certaines inquiétudes :

>     - Peu d'engagement de la QAT dans les cartes controversées.

Reviewer: ```suggestion
    - Peu d'engagement de la QAT dans les maps controversées.
```

    - Accessibilité limitée pour l'entrée de nouveaux utilisateurs dans l'équipe, ce qui entraîne une stagnation.
    - Le rôle de la QAT dans la communauté de mapping n'était pas clair.

> **2749 beatmaps** ont été classés en 2018. La croissance impressionnante de cette année peut être largement attribuée à l'abandon du rôle d'assurance qualité de la QAT et à une plus grande initiative des Beatmap Nominators.

Reviewer: ```suggestion
**2749 beatmaps** ont été classées en 2018. La croissance impressionnante de cette année peut être largement attribuée à l'abandon du rôle d'assurance qualité de la QAT et à une plus grande initiative des Beatmap Nominators.
```

- **29/01/2019 :** Une proposition de réforme de la QAT a été créée à partir d'un conglomérat de propositions d'autres utilisateurs.
  - La [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (*NAT*) a été introduite par cette proposition, et a été mise en œuvre plus tard cette année.

### Mars

- **01/03/2019 :** La [Mappers' Guild](/wiki/Mappers_Guild) est passée d'une inscription sur dossier à une inscription automatique pour toute personne ayant 3 maps classées.
  - Les beatmaps des [Artistes associés](/wiki/Featured_Artists) ont été créées et classées plus fréquemment.
  - Le canal `#mappers-guild` a été créé dans le [serveur Discord osu!dev](/wiki/osu!dev_Discord_server).
- **13/03/2019 :** La règle exigeant que les nominations consécutives soient espacées d'au moins 24 heures a été supprimée.
  - Cette règle a fait l'objet d'une plainte majeure de la part des mappeurs et des Beatmap Nominators, car elle donnait l'impression de faire perdre inutilement du temps à tout le monde. Si un problème était découvert après la première nomination, le deuxième nominateur évitait souvent de le signaler pour éviter une attente supplémentaire de 24 heures.

>   - Parallèlement à ce changement, les Beatmap Nominators peuvent opposer leur veto aux beatmaps qualifiés. Cette règle existait à l'origine parce que les Beatmap Nominators avaient besoin de 24 heures pour appliquer potentiellement des vetos aux maps avec une nomination.

Reviewer: ```suggestion
  - Parallèlement à ce changement, les Beatmap Nominators peuvent opposer leur veto aux beatmaps qualifiées. Cette règle existait à l'origine parce que les Beatmap Nominators avaient besoin de 24 heures pour appliquer potentiellement des vetos aux maps avec une nomination.
```

### Mai

- **05/05/2019 :** La restructuration des BN/QAT a été finalisée.
  - La Quality Assurance Team a été renommée en [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team).
  - Les Beatmap Nominators sont devenus responsables des disqualifications et de la médiation du veto. La médiation du veto a été traitée par un vote à l'aveugle plutôt que par une discussion.
  - Le site des [Beatmap Nominator](https://bn.mappersguild.com/) a été publié pour automatiser de nombreuses tâches des BN/NAT.

>     - Grâce au site web, les utilisateurs pouvaient poser leur candidature à la BN à tout moment, contrairement au système précédent, basé sur des cycles. De ce fait, les Beatmap Nominators ont été ajoutés à des moments irréguliers plutôt que par grandes vagues.

Reviewer: ```suggestion
    - Grâce au site web, les utilisateurs pouvaient poser leur candidature à la BN à tout moment, contrairement au système précédent, basé sur des cycles. De ce fait, les Beatmap Nominators ont été ajoutés par intermitences plutôt que par grandes vagues.
```

>     - Les signalements de beatmaps inappropriés et de comportements répréhensibles des BN/NAT sont passés d'un formulaire Google au site web des BN.

Reviewer: ```suggestion
    - Les signalements de beatmaps inappropriées et de comportements répréhensibles des BN/NAT sont passés d'un formulaire Google au site web des BN.
```

>     - Les statistiques et les évaluations de l'activité et de la compétence du modding et des nominateurs des BN sont passées de Google Spreadsheets, douloureusement inefficaces, au site web des BN, ce qui a considérablement augmenté la productivité de la gestion des Beatmap Nominator.

Reviewer: ```suggestion
    - Les statistiques et les évaluations de l'activité et de la compétence du modding et des nominateurs des BN sont passées de Google Spreadsheets, douloureusement inefficaces, au site web des BN, ce qui a considérablement augmenté la productivité de la gestion des Beatmap Nominators.
```

### Août

- **05/08/2019 :** ![] [flag_SE] [Naxess](https://osu.ppy.sh/users/8129817) a publié l'outil [Mapset Verifier](/wiki/Projects#modding), qui automatise une partie importante des processus de vérification des beatmaps pour les Beatmap Nominators.
- **06/08/2019 :** Les Beatmap Nominators ont pu participer à l'évaluation des demandes des BN.
  - Les contributions des BN sont visibles pour les NAT lorsqu'il s'agit de parvenir à un consensus, bien qu'elles soient rarement utilisées pour influencer un vote.
  - La NAT utilise souvent la contribution des BN pour aider à déterminer les futurs membres du NAT.

### Octobre

- **22/10/2019 :** ![][flag_US] [Ascendance](https://osu.ppy.sh/users/2931883) est devenu un leader du NAT après que ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) se soit retiré.
  - Les dirigeants des NAT ayant souvent été passifs face aux décisions négatives, Ascendance a joué un rôle de contrôle actif et a annoncé les mauvaises nouvelles concernant les suppressions des NAT. <!-- source is discord logs -->

> **3217 beatmaps** ont été classés en 2019.

Reviewer: ```suggestion
**3217 beatmaps** ont été classées en 2019.
```

## 2020

### Février

- **07/02/2020 :** Les premiers badges de profil Elite Nominator ont été attribués pour récompenser les performances exceptionnelles continues des membres de l'équipe des BN.

>   - Elles avaient été prévues à l'origine lors du bouleversement de la QAT deux ans auparavant.

Reviewer: On parle des badges là non ? Je met "Ils" au cas où
```suggestion
  - Ils avaient été prévues à l'origine lors du bouleversement de la QAT deux ans auparavant.
```

- **21/02/2020 :**  La direction de la NAT en tant que concept a été abandonnée lorsque ![][flag_US] [Ascendance](https://osu.ppy.sh/users/2931883) a quitté la direction du NAT.
  - Les tâches spécifiques aux leaders de la NAT ont été déléguées à l'ensemble de l'équipe.
- **24/02/2020 :** Le système [Quality Assurance Helper](/wiki/People/The_Team/Beatmap_Nominators/General_Information#quality-assurance-helpers) a été transféré de Trello au [Beatmap Nominator website](https://bn.mappersguild.com/).

>   - Ce changement a été effectué pour tenir compte de l'activité d'assurance qualité dans les évaluations des Beatmap Nominator.

Reviewer: ```suggestion
  - Ce changement a été effectué pour tenir compte de l'activité d'assurance qualité dans les évaluations des Beatmap Nominators.
```

### Avril

- **11/04/2020 :** Le Mappers' Report, dirigé par ![][flag_DE] [Feerum](https://osu.ppy.sh/users/4815717), a été créé pour résumer les événements de la communauté de mapping par le biais d'articles en première page.

### Octobre

- **02/10/2020 :** Les [Beatmap Nominator Expectations](/wiki/People/The_Team/Beatmap_Nominators/Expectations) ont été établies pour combattre les problèmes de comportement au sein du groupe.
  - Les préoccupations concernaient notamment le manque d'accessibilité, les actions inciviles dans les milieux communautaires et les normes de qualité douteuses en matière de classement.
  - Cela a entraîné le retrait immédiat de 5 Beatmap Nominators.

> - **29/10/2020 :** La compétence du mode de jeu d'un Beatmap Nominator est devenue visible sur le site d'osu!.

Reviewer: ```suggestion
- **29/10/2020 :** La compétence des modes de jeu des Beatmap Nominators sont devenues visibles sur le site d'osu!.
```

### Novembre

> - **30/11/2020 :** Les beatmaps hybrides ont été retravaillés pour nécessiter 2 nominations par mode de jeu.

Reviewer: ```suggestion
- **30/11/2020 :** Les beatmaps hybrides ont été retravaillées pour nécessiter 2 nominations par mode de jeu.
```

  - Les nominateurs compétents dans plusieurs modes de jeu peuvent sélectionner un ou plusieurs modes de jeu à nommer sur les mapsets applicables.

### Décembre

> - **17/12/2020 :** La soumission et l'évaluation de l'examen du contenu inapproprié sont passées des discussions sur Discord aux votes à l'aveugle via le site web des [Beatmap Nominator](https://bn.mappersguild.com/).

Reviewer: ```suggestion
- **17/12/2020 :** La soumission et l'évaluation de l'examen du contenu inapproprié sont passées des discussions sur Discord aux votes à l'aveugle via le site web des [Beatmap Nominators](https://bn.mappersguild.com/).
```
